server/channel_player.go

```diff
@@ -670,7 +670,7 @@ func (server ChannelServer) playerUseInventoryItem(conn mnet.Client, reader mpac
 	slot := reader.ReadInt16()
 	itemid := reader.ReadInt32()
 
-	plr.UseItem(slot, itemid)
+	plr.ApplyUseItem(slot, itemid, server.db)
```

Comment:
I don't think plr needs this function. You could replace this line with the contents of the function as the level of abstraction it provides isn't needed.

Reply:
I agree, it isn't really needed. I removed and amended the commit.